pyEpiabm/run_tests.py

```diff
@@ -0,0 +1,269 @@
+#
+# Runs all unit tests included in pyEpiabm.
+# Run from epiabm directory as `coverage run pyEpiabm/run_tests.py --unit`
+#
+
+from __future__ import absolute_import, division
+from __future__ import print_function, unicode_literals
+import argparse
+import os
+import sys
+import re
+import subprocess
+import unittest
+
+
+def run_unit_tests():
+    """
+    Runs unit tests (without subprocesses).
+    """
+    tests = os.path.join('pyEpiabm', 'pyEpiabm', 'tests')
```

Comment:
```suggestion
    tests = os.path.join(os.path.dirname(os.path.abspath(__file__)), 'pyEpiabm', 'tests')
```
Currently, it's oversensitive to where the run_test.py is called from---this should fix that

Reply:
Ah brilliant - this was frustrating me!